src/Converter/RequestToRefundUnitsConverter.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Converter;
+
+use Symfony\Component\HttpFoundation\Request;
+
+final class RequestToRefundUnitsConverter implements RequestToRefundUnitsConverterInterface
+{
+    public function __construct(private iterable $refundUnitsConverters)
+    {
+    }
+
+    public function convert(Request $request): array
+    {
+        $units = [];
+
+        /** @var RequestToRefundUnitsConverterInterface $refundUnitsConverter */
```

Comment:
We could in theory make it parametrized in the constructor as `/** @param RequestToRefundUnitsConverterInterface[] $refundUnitsConverters */`... I suppose it's the matter of taste, but I prefer comments above the function than in it 😄